awx/main/migrations/0152_instance_node_type.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.20 on 2021-07-26 19:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0151_rename_managed_by_tower'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='instance',
+            name='node_type',
+            field=models.CharField(
+                choices=[('control', 'Control plane node'), ('execution', 'Execution plane node'), ('hybrid', 'Controller and execution')],
```

Comment:
hybrid, it's now a thing. Let it be known!

Reply:
it's better on the gas mileage.